crates/chia-consensus/src/gen/opcodes.rs

```diff
@@ -64,6 +64,10 @@ pub const SOFTFORK: ConditionOpcode = 90;
 
 pub const CREATE_COIN_COST: Cost = 1_800_000;
 pub const AGG_SIG_COST: Cost = 1_200_000;
+
+pub const GENERIC_CONDITION_COST: Cost = 1_200_000;
```

Comment:
I think this should be in the range 100-500.

Reply:
Set to 500